test-data/unit/check-newsyntax.test

```diff
@@ -142,4 +142,13 @@ value = 10.5142
 width = 10
 precision = 4
 f'result: {value:{width}.{precision}}'
-[builtins fixtures/primitives.pyi]
+[builtins fixtures/f_string.pyi]
+
+[case testNewSyntaxFStringSingleField]
+# flags: --python-version 3.6
+v = 1
+f'{v}' + ''
```

Comment:
I'd write `reveal_type(f'{v}')  # E: Revealed type is 'builtins.str'`, which tests more explicitly what you want to achieve.